internal/pkg/manifest/validate.go

```diff
@@ -1180,6 +1185,9 @@ func validateContainerDeps(opts validateDependenciesOpts) error {
 		dependsOn:   opts.imageConfig.DependsOn,
 		isEssential: true,
 	}
+	if !opts.logging.IsEmpty() {
+		containerDependencies[firelensContainerName] = containerDependency{}
```

Comment:
so it's not possible for the firelens container to depend on another one?

Reply:
Right now it's not configurable and it is always an essential one.